modules/snippy/core/main.nf

```diff
@@ -0,0 +1,53 @@
+process SNIPPY_CORE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::snippy=4.6.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/snippy:4.6.0--hdfd78af_2':
+        'quay.io/biocontainers/snippy:4.6.0--hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(vcf), path(aligned_fa)
+    path reference
+
+    output:
+    tuple val(meta), path("${prefix}.aln")     , emit: aln
+    tuple val(meta), path("${prefix}.full.aln"), emit: full_aln
+    tuple val(meta), path("${prefix}.tab")     , emit: tab
+    tuple val(meta), path("${prefix}.vcf")     , emit: vcf
+    tuple val(meta), path("${prefix}.txt")     , emit: txt
+    path "versions.yml"                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
```suggestion
    def prefix = task.ext.prefix ?: "${meta.id}"
```

Reply:
will this be ok if I'm using `$prefix` in the output block?

Reply:
Correct me if I'm wrong, but the output prefix won't be necessary as you shouldn't have any 'input' fasta/aln/tab etc. in your main directory, correct? You could just give `*fasta` `*aln` directly? Or does the output also go in `samples`?

Reply:
Think we'll have an issue with `*.aln` capturing both `${prefix}.aln` and `${prefix}.full.aln` ?

Reply:
Ah fair point. Fine as is then!